resources/src/topicHeader.cpp

```diff
@@ -49,6 +49,7 @@ const char* CONTROL_LOADPARAM="/kraken/control/loadparam_srv";
 
 //sensor topics
 const char* SENSOR_IMU="/kraken/sensor/imu";
+const char* SENSOR_IMU_NEW="/kraken/sensor/imu_new";
```

Comment:
@icyflame : My bad! Forgot to push the updated `manifest.xml` file. Test now.

![image](https://cloud.githubusercontent.com/assets/17109060/18031580/e6353790-6d03-11e6-9726-a0f19e307a95.png)